AsyncDisplayKit/Layout/ASLayoutElement.h

```diff
@@ -363,4 +364,10 @@ extern NSString * const ASLayoutElementStyleLayoutPositionProperty;
 
 @end
 
+@protocol ASLayoutElementStylability
+
+- (id<ASLayoutElement>)styleWithBlock:(void (^)(__kindof ASLayoutElementStyle *style))styleBlock;
```

Comment:
What do you think of the name `styledWithBlock`? That emphasizes the fact that the method has a return value.